Project_2/djangoProject/cars/views.py

```diff
@@ -38,7 +51,16 @@ def more_details(request, pk):
     car_queryset = Car.objects.filter(pk=pk)
     try:
         car = car_queryset.first()
+        if not car.described:
+            get_car_detail = car_queryset.values(*get_detail_list)[0]
+            api_data = api_detail(car.vin)
```

Comment:
This can give some error, especially if the API does not respond with a 200 status code. Add this line in a try-except block and show a message that the API call was not successful, if there was an error